lang/fr.json

```diff
@@ -146,10 +146,14 @@
     "VTM5E.RollingWillpower": "Jet de Volonté",
     "VTM5E.RousingBlood": "Exaltation du sang",
     "VTM5E.CriticalSuccess": "Succès critique !",
+    "VTM5E.Success": "Succès!",
     "VTM5E.Successes": "Succès",
+    "VTM5E.Fail": "Échouer!",
     "VTM5E.MessyCritical": "Echec critique !",
+    "VTM5E.BestialFailure": "Succès Triomphal !",
```

Comment:
I don't think that's the right translation
I'm not fluent in French but should be something around "échec bestial"
Best check the french version of the V5 corebook